### PR TITLE
updatecli: link to the original Pull Request

### PR DESCRIPTION
**Pull request by v1v**
## What does this PR do?

Add the original Pull Request that contains the changes in the specs to be in the description of the Pull Request created with the updatecli automation. This will help with track what APM Agents use it.

## Checklist

- [ ] This is an enhancement of existing features, or a new feature in existing plugins
  - [ ] I have updated [CHANGELOG.asciidoc](https://github.com/elastic/apm-agent-java/blob/main/CHANGELOG.asciidoc)
  - [ ] I have added tests that prove my fix is effective or that my feature works
  - [ ] Added an API method or config option? Document in which version this will be introduced
  - [ ] I have made corresponding changes to the documentation
- [ ] This is a bugfix
  - [ ] I have updated [CHANGELOG.asciidoc](https://github.com/elastic/apm-agent-java/blob/main/CHANGELOG.asciidoc)
  - [ ] I have added tests that would fail without this fix
- [ ] This is a new plugin
  - [ ] I have updated [CHANGELOG.asciidoc](https://github.com/elastic/apm-agent-java/blob/main/CHANGELOG.asciidoc)
  - [ ] My code follows the [style guidelines of this project](https://github.com/elastic/apm-agent-java/blob/main/CONTRIBUTING.md#java-language-formatting-guidelines)
  - [ ] I have made corresponding changes to the documentation
  - [ ] I have added tests that prove my fix is effective or that my feature works
  - [ ] New and existing [**unit** tests](https://github.com/elastic/apm-agent-java/blob/main/CONTRIBUTING.md#testing) pass locally with my changes
  - [ ] I have updated [supported-technologies.asciidoc](https://github.com/elastic/apm-agent-java/blob/main/docs/supported-technologies.asciidoc)
  - [ ] Added an API method or config option? Document in which version this will be introduced
  - [ ] Added an instrumentation plugin? Describe how you made sure that old, non-supported versions are not instrumented by accident.
- [ ] This is something else
  - [ ] I have updated [CHANGELOG.asciidoc](https://github.com/elastic/apm-agent-java/blob/main/CHANGELOG.asciidoc)

## Test

Given the updatecli and gh cli

When running

```bash
$ GIT_USER=foo \
GIT_EMAIL=1 \
GITHUB_TOKEN=*** \
updatecli apply --config .ci/updatecli.d/update-gherkin-specs.yml --push=false --debug
```

Then

```
...
✔ content: found from file "https://github.com/elastic/apm/commit/main.patch":
From 94b8365d8400b6c119ebef3845e95109c5c5462e 
...
----
https://github.com/elastic/apm/pull/830
----
...
```

### CODE REVIEW

**Review by reakaleek — DISMISSED** [review]

File: .ci/updatecli.d/update-gherkin-specs.yml [hunk 1]
```diff
@@ -62,10 +70,13 @@ actions:
       description: |-
         ### What
         APM agent Gherkin specs automatic sync
+
         ### Why
         *Changeset*
+        * {{ source "pull_request" }}
```
Comment (SylvainJuge) [inline .ci/updatecli.d/update-gherkin-specs.yml:76]: [question] does this means "get the output of the `pull_request` item above" ? If yes, then that means just injecting a URL link to the PR.

File: .ci/updatecli.d/update-specs.yml [hunk 2]
```diff
@@ -58,9 +67,11 @@ actions:
       description: |-
         ### What
         APM agent json schema automatic sync
+
         ### Why
         *Changeset*
-        * https://github.com/elastic/apm-data/commit/{{ source "sha" }}
+        * {{ source "pull_request" }}
+        * https://github.com/elastic/apm/commit/{{ source "sha" }}
```
Comment (SylvainJuge) [inline .ci/updatecli.d/update-specs.yml:74]: Is that intentional that we move from `apm-data` to `apm` here ?

Comment (v1v) [reply .ci/updatecli.d/update-specs.yml:74]:
```suggestion
        * https://github.com/elastic/apm-data/commit/{{ source "sha" }}
```

Comment (v1v) [reply .ci/updatecli.d/update-specs.yml]: good catch! my bad!

**Review by SylvainJuge — APPROVED** [review]

**Pull request merged**